jobs/designer.md

```diff
@@ -0,0 +1,64 @@
+- Fluency in English (writing and speaking) for daily use.
+- Understanding of the principles of building and managing the team.
+- Proactivity: ability to see points for improvement and initiativeness in proposing changes.
+- Versatileness: you’ll work on the client’s project, but sometimes we can ask you to help the team with internal tasks.
+- Interest in working with AI tools.
+
+## Would be a plus:
+- Knowledge of UI Kit, React, HTML.
+
+
+## What do we offer:
+- Transparent work culture.
+- 50% compensation for self-development activities: courses, conferences, books, educational platforms, English, and sports.
+- Remote work and flexible schedule.
+- 32 paid days off per year – you can use them for vacations, occasional days off, and public holidays, only workdays count.
+- Full coverage of sick leaves.
+- 200 USD per year for medical expenses.
+- 500 USD per year for home office equipment.
+  
+
+## About us
+
+[![datarockets team](https://github.com/datarockets/career/blob/master/images/photo%20from%20New%20Year%20party.3.jpg)](https://www.instagram.com/datarockets/)
+
+
+datarockets is a [remote](https://github.com/datarockets/career#remote) software development company focusing on setting up transparent and honest processes with the team and clients. We adore flexibility, freedom and independence, so we have no micromanagement and draining bureaucracy. 
+At datarockets, the medium project duration is 12 months. With most of the clients, we work for years. Being a relatively small company, we have the freedom to pick the projects that we feel most passionate about, which benefits both our clients and us. We have experience building apps serving a variety of industries, such as FinTech, IoT, Health & Fitness, e-commerce, and many more. Among our clients: YC-backed startups, Individual Entrepreneurs, Mid-sized companies, Public Institutions, and Series E startups.
+
+Check out our resources for more information:
+
+- [Case studies](https://datarockets.com/case-studies/) to learn more about the types of projects we work with.
+- [Company Codex](https://github.com/datarockets/career#work-as-a-team-with-clients) to see if we share the same values.
+- [Career page](https://datarockets.com/career/) to learn more about our culture and processes.
+
+
+## How to apply 💌
+
+To apply, please, send us an email to [join@datarockets.com](mailto:join@datarockets.com) with your CV. 
+As an option, you can send your application via our [Career page](https://datarockets.com/career/) on a website. 
+
+Also, we are curious about your thoughts on the questions below, please, add your answers to the cover letter.
+
+- ???
+- ???
+
+Due to a big number of applications, we leave a right for us not to consider applications without answers to our questions. Please, note that the time of receiving a reply email may increase to one-two week.
```

Comment:
There is no questions mentioned here and I'm not sure I have any quick ones. As for designer, I'd like them to attach their portfolio. Can you incorporate that?

Reply:
Yep, sure thing, will add! Thank you for your review!